anndata/_core/aligned_mapping.py

```diff
@@ -84,7 +94,10 @@ def parent(self) -> Union["anndata.AnnData", "raw.Raw"]:
     def copy(self):
         d = self._actual_class(self.parent, self._axis)
         for k, v in self.items():
-            d[k] = v.copy()
+            if isinstance(v, AwkArray):
+                d[k] = copy(v)
```

Comment:
```suggestion
            if isinstance(v, AwkArray):
                # Shallow copy since awkward array buffers are immutable
                d[k] = copy(v)
```